src/components/removeHotel/RemoveHotel.jsx

```diff
@@ -55,43 +56,42 @@ const RemoveHotel = () => {
           Ups! Something went wrong
         </div>
       )}
-      <header>
-        <h2 className="mt-10 mb-10 font-Taxicab text-3xl text-gray-800 md:mt-[25vh]">
-          Delete Hotel
-        </h2>
-      </header>
-      <table className="mx-auto w-4/5 text-gray-500 md:table-auto">
-        <thead className="bg-gray-500 font-Taxicab text-lg uppercase text-slate-50 dark:bg-gray-700 dark:text-gray-400">
-          <tr>
-            <th className="px-6 py-3 text-left">Item</th>
-            <th className="w-96 py-3 px-6 text-left">Hotel name</th>
-            <th className="py-3 px-6 ">Action</th>
-          </tr>
-        </thead>
-        <tbody className="font-Metrophobic">
-          {hotels.map((hotel, index) => (
-            <tr key={index} className="even:bg-white">
-              <td className="px-6 py-4">{index + 1}</td>
-              <td className="px-6 py-4 text-neutral-900">{hotel.name}</td>
-              <td className="flex items-center justify-center py-4">
-                <button
-                  className="rounded-md bg-red-500 px-2 py-1 text-white hover:bg-red-200 hover:text-red-800"
-                  onClick={() => handleClick(hotel)}
-                >
-                  <DeleteForeverIcon />
-                </button>
-              </td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
-      <ConfirmDelete
-        visible={modalVisible}
-        onOk={handleOk}
-        onCancel={handleCancel}
-        hotel={modalInfo}
-      />
-    </section>
+      <section className="flex h-screen">
+        <div className="flex flex-col items-center gap-3 p-9">
+          <img className="w-4/12" src="Hotelzilla-logo.png" alt="" />
+          <h2 className="text-2xl capitalize">Delete hotel</h2>
+          <table className="h-[40vh] overflow-y-auto overscroll-contain">
+            <thead className="font-Taxicab uppercase">
+              <tr>
+                <th className="">Item</th>
+                <th className="">Hotel name</th>
+                <th className="">Action</th>
+              </tr>
+            </thead>
+            <tbody className="h-[40vh] ">
+              {hotels.map((hotel, index) => (
+                <tr key={index} className="even:bg-white">
+                  <td className="py-3 px-6">{index + 1}</td>
+                  <td className="py-3 px-6">{hotel.name}</td>
+                  <td className="py-3 px-6">
+                    <button className="" onClick={() => handleClick(hotel)}>
+                      <DeleteForeverIcon />
+                    </button>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+          <ConfirmDelete
+            visible={modalVisible}
+            onOk={handleOk}
+            onCancel={handleCancel}
+            hotel={modalInfo}
+          />
+        </div>
+      </section>
+      <BackButton />
+    </>
```

Comment:
- [ ] Kindly make the logo icon disappear in the `desktop version` for the delete hotel and the form in the middle of the page to match the rest of the forms.
![image](https://user-images.githubusercontent.com/24703171/168716952-2dd1bea8-8561-428b-a85a-4bbca9e065da.png)